### ISSUE TITLE
Flowers Classification using DL

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#252]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Flowers Classification</br>
:red_circle: **Aim** : Build a model based on Tensorflow to classify the flowers from the given dataset.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/ryanholbrook/tensorflow-flowers</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
```
Full Name:- B Mani Chander
Github Profile link:- https://github.com/manichander429
Email id:- [manichander429@gmail.com](mailto:manichander429@gmail.com)
Approach for this project:- so recently I developed a cnn model for diabetic retinopathy detection. Coming to the approach First, I will perform data augmentation, so that the model I develop can learn enough features to detect. Then, I would try using different models/architectures like cnn, ResNet, AlexNet, GoogleNet.. Etc. Then I will perform some hypertuning in order to find the optimal hyperparameters and then I will compare the accuracy of the developed models.
What is your participant role?-contributor ,SSOC-Season-2
```

Hi @manichander429 nice to have you here. I like your approach for solving this issue. This issue will be assigned to you once the program starts officially.

Till then, star the repository and explore more in open source.

**Comment by Manoj-2702** [comment]
Full Name: Manoj Kumar H S
Git Profile Link: [https://github.com/Manoj-2702](https://github.com/Manoj-2702)
Email ID: hsmanojkumar2003@gmail.com
Approach: So firstly I would perform data augmentation techniques such as random rotations, flips, zooms, and shifts to increase the diversity and quantity of training data. Then I would try using CNN Architectures such as VGG16, ResNet50, or InceptionV3, without the top classification layers. Later on add custom fully connected layers on top of the base model to adapt it to the flower classification task. Then, compile the model with an appropriate optimizer (e.g., Adam) and a suitable loss function (e.g., categorical cross-entropy). Hypertune the model to get better accuracy.
What is your participant Role? Contributor, SSOC-Season-2

**Comment by abhisheks008** [comment]
Hi @Manoj-2702 nice to have you here. This issue is already pre booked by someone else, you can pick up other issues.

**Comment by abhisheks008** [comment]
Hi @manichander429 would you like to take this issue?

**Comment by manichander429** [comment]
Yaa i am more than interested

**Comment by manichander429** [comment]
@abhisheks008 Yaa I am more than interested

**Comment by abhisheks008** [comment]
This issue is assigned to you @manichander429

**Comment by YashSachan2** [comment]
@abhisheks008 Can I take on this issue under Codepeak-23.I plan to compare several models like YOLO,Resnet,Mobilenet preprocessing based on the requirement of each model and then selecting the best out of them.

**Comment by abhisheks008** [comment]
> @abhisheks008 Can I take on this issue under Codepeak-23.I plan to compare several models like YOLO,Resnet,Mobilenet preprocessing based on the requirement of each model and then selecting the best out of them.

This issue will be assigned to you, once you finish the previous one.

**Comment by YashSachan2** [comment]
@abhisheks008 Can you assign me now under Codepeak-23?

**Comment by abhisheks008** [comment]
Hi @YashSachan2 this issue will be assigned to you, once you finish the previous one.

**Comment by abhisheks008** [comment]
Issue assigned to you @YashSachan2

**Comment by YashSachan2** [comment]
@abhisheks008 I have made the pr for flower classification.Please look into it.
